src/components/LandingPages/WorkerLanding.tsx

```diff
@@ -183,85 +207,108 @@ class WorkerLanding extends Component<Props, State> {
       (client, i) => (
-        <div key={client.username} className="card mb-3">
-          <div className="card-body">
-            <div className="d-flex flex-row">
-              <div className="d-flex flex-column mr-4">
-                <img
-                  alt="a blank profile"
-                  className="profile-picture"
-                  src={GenericProfilePicture}
-                />
-              </div>
-              <div className="d-flex flex-lg-column mr-4">
-                <h5 className="card-title mb-3 h4">
-                  {client.firstName}
-
-                  {client.lastName}
-                </h5>
-                <h6 className="card-subtitle mb-2 text-muted">
-                  {client.username}
-                </h6>
-                <h6 className="card-subtitle mb-2 text-muted">
-                  {client.email}
-                </h6>
-                <h6 className="card-subtitle mb-2 text-muted">
-                  #
-{client.phone}
-                </h6>
-                <h6 className="card-subtitle mb-2 text-muted">
-                  {client.address}
-                </h6>
-                <h6 className="card-subtitle mb-2 text-muted">
-                  {client.city}
-                  {', '}
-                  {client.state}
-
-                  {client.zipcode}
-                </h6>
-                <p className="card-text">
-                  Some information about the client here.
-                </p>
-                <Link to={`/profile/${client.username}`}>
-                  <button type="button" className="btn btn-primary">
-                    Client Profile
-                  </button>
-                </Link>
-              </div>
-              <div className="d-flex flex-column mr-4">
-                <h5 className="card-title">Client Actions</h5>
-                <button
-                  type="button"
-                  className="btn btn-success mb-2 btn-sm"
-                  onClick={(event) =>
-                    this.handleClickUploadDocuments(event, client)
-                  }
-                >
-                  Upload Document
-                </button>
-                <button
-                  type="button"
-                  className="btn btn-danger mb-2 btn-sm"
-                  onClick={(event) =>
-                    this.handleClickViewDocuments(event, client)
-                  }
-                >
-                  View Documents
-                </button>
-                <button
-                  type="button"
-                  className="btn btn-dark mb-2 btn-sm"
-                  onClick={(event) =>
-                    this.handleClickSendApplication(event, client)
-                  }
-                >
-                  Send Application
-                </button>
+        <div key={client.username} className="card client-card mb-4 mr-4 flex-column">
+          <div className="dropdown" style={{ position: 'absolute', top: 24, right: 24 }}>
```

Comment:
if possible, lets not have in-line styling inside the html. If needed, we can define new CSS classes and have them apply to a bunch of areas at once.